tests/realm.cpp

```diff
@@ -1290,6 +1290,22 @@ TEST_CASE("SharedRealm: compact on launch") {
             REQUIRE(num_opens == 3);
         }).join();
     }
+
+    SECTION("multiple versions when multiple threads") {
```

Comment:
I would argue this is the wrong place to test the core function. That must be assumed or validated that it's tested in core.

Reply:
True, but I simply wish to be sure that I am able to get a value. Otherwise it feels a bit speculative.